cpp/include/raft/random/rmat_rectangular_generator.cuh

```diff
@@ -65,6 +68,58 @@ namespace raft::random {
  * @{
  */
 template <typename IdxT, typename ProbT>
+void rmat_rectangular_gen(const raft::handle_t& handle,
+                          raft::random::RngState& r,
+                          raft::device_vector_view<const ProbT, IdxT> theta,
+                          raft::device_vector_view<IdxT, IdxT> out,
+                          raft::device_vector_view<IdxT, IdxT> out_src,
+                          raft::device_vector_view<IdxT, IdxT> out_dst,
+                          IdxT r_scale,
+                          IdxT c_scale,
+                          IdxT n_edges)
+{
+  detail::rmat_rectangular_gen_caller(out.data_handle(),
```

Comment:
Are we able to validate sizes here?  For example, since the output arrays represent an edge list, can we validate that they are all the same size?

Reply:
I've added size validation; thanks!